### PR TITLE
Added classes to elements with only an ID

### PR DESCRIPTION
**Pull request by DeathCamel57**
Added to #scroll__here, #index__tree, #draft__status, #tool__bar, #dw__toc. This in in re: #2172

### CODE REVIEW

**Comment by Klap-in** [comment]
The #scroll__here id is used for moving to some point at the page intentionally.
See also https://github.com/splitbrain/dokuwiki/blob/master/lib/scripts/behaviour.js#L68

So it a functional id, and in my view it has no meaning for layouting. Is it worth adding a class?

**Comment by krisak** [comment]
I dont believe that it is worth adding a class. It just polutes DOM for no reason.

**Comment by DeathCamel57** [comment]
#scroll__here still has the id, but I used the class

**Comment by DeathCamel57** [comment]
I had a brain fart. I meant removed, not used.

File: inc/html.php [hunk 1]
```diff
@@ -270,7 +270,7 @@ function html_show($txt=null){
 
     if (!is_null($txt)){
         //PreviewHeader
-        echo '<br id="scroll__here" />';
+        echo '<br id="scroll__here"/>';
```
Comment (splitbrain) [inline inc/html.php]: can you re-add the blank before the self-close please?

File: inc/html.php [hunk 2]
```diff
@@ -1002,7 +1002,7 @@ function html_li_index($item){
         // scroll to the current item
         if($item['id'] == $INFO['id'] && $ACT == 'index') {
             $id = ' id="scroll__here"';
-            $class = ' bounce';
+            $class = ' scroll_here bounce';
```
Comment (splitbrain) [inline inc/html.php]: scroll_here should not be a class here

**Comment by DeathCamel57** [comment]
Made then changes that you requested @splitbrain

**Comment by DeathCamel57** [comment]
Whoops, I didn't mean to close it!

**Pull request merged**